pkg/executor/importer/chunk_process.go

```diff
@@ -100,8 +101,14 @@ func (b *deliverKVBatch) add(kvs *kv.Pairs) {
 	}
 }
 
-// chunkEncoder encode data chunk(either a data file or part of a file).
-type chunkEncoder struct {
+type chunkEncoder interface {
+	init() error
+	encodeLoop(ctx context.Context) error
+	logFields() []zap.Field
```

Comment:
seems it's only used when task ends. In order to not let developer use it when task is running, I prefer we name it `summary`, `summaryFields` or something.